v1.0/docs/admin/cluster-components.md

```diff
@@ -1,5 +1,6 @@
 ---
 layout: docwithnav
+title: Kubernetes Cluster Admin Guide: Cluster Components
```

Comment:
For Richie or someone - we should get consistency with titles.  Some say "Kubernetes Cluster Admin Guide: Foo" and other just "Foo"